pkg/mcs/resource_manager/server/grpc_service.go

```diff
@@ -33,8 +33,8 @@ import (
 )
 
 var (
-	// errNotLeader is returned when current server is not the leader.
-	errNotLeader = status.Errorf(codes.Unavailable, "not leader")
+	// errNotServing is returned when current server is not serving.
```

Comment:
Will it affect the PD-only mode?

Reply:
I am not sure. cc @JmPotato

Reply:
I think we need to keep it since PD still needs the error message.

Reply:
ok